MediaBrowser.Model/Branding/BrandingOptions.cs

```diff
@@ -1,19 +1,38 @@
-#pragma warning disable CS1591
+using System.Text.Json.Serialization;
+using System.Xml.Serialization;
 
-namespace MediaBrowser.Model.Branding
+namespace MediaBrowser.Model.Branding;
+
+/// <summary>
+/// The branding options.
+/// </summary>
+public class BrandingOptions
 {
-    public class BrandingOptions
-    {
-        /// <summary>
-        /// Gets or sets the login disclaimer.
-        /// </summary>
-        /// <value>The login disclaimer.</value>
-        public string? LoginDisclaimer { get; set; }
+    /// <summary>
+    /// Gets or sets the login disclaimer.
+    /// </summary>
+    /// <value>The login disclaimer.</value>
+    public string? LoginDisclaimer { get; set; }
+
+    /// <summary>
+    /// Gets or sets the custom CSS.
+    /// </summary>
+    /// <value>The custom CSS.</value>
+    public string? CustomCss { get; set; }
+
+    /// <summary>
+    /// Gets or sets the splashscreen location on disk.
+    /// </summary>
+    /// <remarks>
+    /// Not served via the API.
+    /// Only used to save the custom uploaded user splashscreen in the configuration file.
+    /// </remarks>
+    [JsonIgnore]
+    public string? SplashscreenLocation { get; set; }
 
-        /// <summary>
-        /// Gets or sets the custom CSS.
-        /// </summary>
-        /// <value>The custom CSS.</value>
-        public string? CustomCss { get; set; }
-    }
+    /// <summary>
+    /// Gets the splashscreen url.
+    /// </summary>
+    [XmlIgnore]
+    public string SplashscreenUrl => "/Branding/Splashscreen";
```

Comment:
Why?

Reply:
This was requested above, but I think we can just remove it